julia/src/model.jl

```diff
@@ -6,15 +6,19 @@ mutable struct StanModelStruct end
 
 A StanModel instance encapsulates a Stan model instantiated with data.
```

Comment:
Is this the standard way to do Julia doc, with a line for the class doc and then a line for the constructor?  It looks very strange.

Reply:
As far as I know and see.  Agreed, it looks strange.